server/utils/applications/forPagesInTask.ts

```diff
@@ -0,0 +1,27 @@
+import { FormArtifact, UiTask } from '../../@types/ui'
+import { getPage } from './getPage'
+import TaskListPage from '../../form-pages/taskListPage'
+
+export const forPagesInTask = (
```

Comment:
could we give this function a clearer name? I'm finding it quite confusing - the functionality is complex and the name isn't guiding me :)